File: test_dir/python/src/strings/escape_characters.py
def string_esc_chars_fn():
    a: str = "\n"
    b: str = "abcd\n"
    c: str = "\t"
    d: str = "abcd\t"
    e: str = "\r"
    f: str = "abcd\r"
    g: str = "\b"
    h: str = "abcd\b"
    i: str = "\f"
    j: str = "abcd\f"
    k: str = "\\"
    l1: str = "abcd\\"
    print(a + b + c + d + e + f + g + h + i + j + k + l1)
    m: str = "abcd\\\\"
    print(m)

File: test_dir/cpp/src/strings/escape_characters.cpp
#include "strings\escape_characters.h"

void string_esc_chars_fn() {
    PyStr a = PyStr("\n");
    PyStr b = PyStr("abcd\n");
    PyStr c = PyStr("\t");
    PyStr d = PyStr("abcd\t");
    PyStr e = PyStr("\r");
    PyStr f = PyStr("abcd\r");
    PyStr g = PyStr("\b");
    PyStr h = PyStr("abcd\b");
    PyStr i = PyStr("\f");
    PyStr j = PyStr("abcd\f");
    PyStr k = PyStr("\\");
    PyStr l1 = PyStr("abcd\\");
    (((((((((((a + b) + c) + d) + e) + f) + g) + h) + i) + j) + k) + l1)
        .print();
    PyStr m = PyStr("abcd\\\\");
    m.print();
}